test/cpp/qps/BUILD

```diff
@@ -165,6 +165,18 @@ grpc_cc_test(
 )
 
 grpc_cc_test(
+    name = "qps_openloop_600_channel_300B_req_50B_resp_test",
```

Comment:
why is there a separate .cc for this?

Reply:
shouldn't it be part of scenario_config.py?

Reply:
Thanks for the review! Created #12987 for adding this scenario in `scenario_config.py`. Will close this PR.